julia-package/brainflow/src/data_filter.jl

```diff
@@ -67,15 +73,29 @@ end
     return
 end
 
-@brainflow_rethrow function perform_bandpass(data, sampling_rate::Integer, center_freq::Float64,
-    band_width::Float64, order::Integer, filter_type::Integer, ripple::Float64)
+@brainflow_rethrow function perform_bandpass(
+    data, 
+    sampling_rate::Integer, 
+    center_freq::Float64,
+    band_width::Float64, 
+    order::Integer, 
+    filter_type::Integer, 
+    ripple::Float64
+)
     ccall((:perform_bandpass, DATA_HANDLER_INTERFACE), Cint, (Ptr{Float64}, Cint, Cint, Float64, Float64, Cint, Cint, Float64),
             data, length(data), Int32(sampling_rate), Float64(center_freq), Float64(band_width), Int32(order), Int32(filter_type), Float64(ripple))
     return
 end
 
-@brainflow_rethrow function perform_bandstop(data, sampling_rate::Integer, center_freq::Float64,
-    band_width::Float64, order::Integer, filter_type::Integer, ripple::Float64)
+@brainflow_rethrow function perform_bandstop(
+    data, 
+    sampling_rate::Integer,
+    center_freq::Float64 = 50.0, #Hz
```

Comment:
lets remove defaults here, for USA there must be different values and lets keep it up to the user to provide it

Reply:
Hmm... I had to spend time googling and asking you to figure out good defaults.
Is there a way to add them to the docs somehow?

Reply:
maybe we can add methods like `perform_notch_eu` or `perform_notch_usa` to all languages?

Reply:
I guess that's valid solution as well.